.github/workflows/tests_linux_x86_mpi_gpu.yml

```diff
@@ -222,16 +221,11 @@ jobs:
           echo "PIP Path => $pip_path"
           echo "pip=$pip_path" >> $GITHUB_OUTPUT
 
-      - name: Install Latest PennyLane
-        # We want to install the latest PL on non workflow_call events
-        if: inputs.pennylane-version == 'latest'  || inputs.pennylane-version == ''
-        run: python -m pip install git+https://github.com/PennyLaneAI/pennylane.git@master
-
       - name: Install required packages
         run: |
           source /etc/profile.d/modules.sh && module use /opt/modules/ && module load ${{ matrix.mpilib }}
-          python -m pip install pip~=22.0
-          python -m pip install ninja cmake custatevec-cu11 pytest pytest-mock flaky pytest-cov mpi4py openfermionpyscf
+          python -m pip install -r requirements-dev.txt
+          python -m pip install cmake custatevec-cu11 mpi4py openfermionpyscf
```

Comment:
Do we also need ninja here? If no, shouldn't cmake be part of the req-dev too?

Reply:
ninja is part of `requirements-dev`. `cmake` is added to `requirements-dev`.